keyboards/duck/tcv3/info.json

```
@@ -0,0 +1,16 @@
+{
+    "keyboard_name": "TCV3", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "width": 19.5, 
+    "height": 6.75, 
```

Removing 0.25u of gap between F-row and number row because I'm that guy.

```suggestion
    "height": 6.5, 
```